kontroller/videoservice.cpp

```diff
@@ -266,7 +266,7 @@ void VideoService::refresh_collection()
 		file.setIcon("movies");
 		files_.push_back(file);
 		file = File();
-		file.setLabel(tr("TV Shows"));
+		file.setLabel(tr("TV shows"));
```

Comment:
Is this a menu headline somewhere?

Reply:
Yes, this shows up on the « Video » page, in the Sources.